.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,16 @@
-## Background
-<!-- What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary -->
+## Description of change
+<!-- Please include a description of the change. What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary. This could include dependencies introduced, changes in behavior, pointers to more detailed documentation -->
 
-## Definition of Done
+## Testing done
+<!-- Please describe testing done to verify the changes. -->
 
-#### Unique to this PR
+## Testing planned
+<!-- Please describe testing planned. -->
+
+## Acceptance Criteria (Definition of Done)
 
+#### Unique to this PR
+<!-- This would be a good place to include feature flag check item and info, specific dashboards and instrumentation check item and info -->
 - [ ] _Replace this line with individual tasks unique to your PR_
 
 #### Applies to all PRs
```

Comment:
With testing being covered above, should we remove the line below or change it to "Appropriate logging"?

Reply:
Updated the checklist item to be "Appropriate logging"